docs/business.yaml

```diff
@@ -3700,31 +3877,27 @@ components:
             streetAddress: 5-4761 Bay Street
             streetAddressAdditional: Building A
             addressCity: Victoria
-            addressRegion: 'BC for British Columbia '
+            addressRegion: 'BC for British Columbia'
             addressCountry: Canada
             postalCode: V8R 2P1
             deliveryInstructions: '"Our apartment is located at the back of the building."'
           deliveryAddress:
             streetAddress: 5-4761 Bay Street
             streetAddressAdditional: Building A
             addressCity: Victoria
-            addressRegion: 'BC for British Columbia '
+            addressRegion: 'BC for British Columbia'
             addressCountry: Canada
             postalCode: V8R 2P1
             deliveryInstructions: '"Our apartment is located at the back of the building."'
     party:
       type: object
       title: party
-      description: 'Represent a person and its role in relation to a business. '
+      description: 'Represent a person and its role in relation to a business.'
       properties:
         party:
           type: object
-          description: 'Represent a person and its role in relation to a business. '
+          description: 'Represent a person and its role in relation to a business.'
```

Comment:
Also, thinking about if the party schema is referenced elsewhere, should we also update the corresponding examples to reflect this change?

<img width="954" alt="image" src="https://github.com/user-attachments/assets/4b79a54a-efbb-4c8d-8aed-7fd1ca3f01cb">

Reply:
Good point, I removed them since it's not in the Business Schemas, also not in the returned responses.
I'm not sure about removing from the x-examples, but we are not really using them and they don't actually show in the specs. 
Any thoughts on this? @argush3

Reply:
I think you can remove them 🤔

Reply:
Removed all `taxId` fields in x-examples under `Party` and `Officer`/`Person`
We can always add them back, if we need them in the future